examples/compute-dg-api.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ranked_df = absolute_df.sort_values(by="DG (kcal/mol)", ascending=True)
</pre><p>Do we have the ability to switch between afinity types? I would guess no, so maybe an issue for the future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Do you mean units? If so this is work slated for later</p>